src/main/java/fr/foodlimit/api/scheduled/ScheduledTasks.java

```diff
@@ -0,0 +1,81 @@
+package fr.foodlimit.api.scheduled;
+
+import fr.foodlimit.api.food.FoodService;
+import fr.foodlimit.api.shared.models.Food;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.tomcat.jni.Local;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.tomcat.jni.Local'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)